sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj

```diff
@@ -28,6 +28,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Amqp" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
+    <PackageReference Include="NETStandard.Library" VersionOverride="2.0.3"/>
```

Comment:
Why is this necessary?

Reply:
Same as above. Assembly resolution for this with current dependency set was coming out to an older version causing build errors and wanting us to explicitly set the assembly version here.

Reply:
This also already exists in the packages file for T1 dependencies. I don't understand the value in overriding locally:
https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L341